core/templates/dev/head/player/PlayerServices.js

```diff
@@ -535,69 +542,76 @@ oppia.factory('oppiaPlayerService', [
           answer, currentInteraction.id, currentInteraction.customization_args);
       }
     },
-    submitAnswer: function(answer, successCallback, delayParamUpdates) {
+    getClassificationResult: function(answer) {
```

Comment:
Should make clear that this returns a promise -- probably via docstring, and maybe call it getClassificationResultAsync().